packages/fondue/src/components/RichTextEditor/Plugins/EmojiPlugin/EmojiPicker/EmojiPickerContent.tsx

```diff
@@ -0,0 +1,158 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { memo, useCallback } from 'react';
+import { Emoji, EmojiCategoryList, EmojiSettings, GridRow, UseEmojiPickerType } from '@udecode/plate-emoji';
+
+export type EmojiPickerContentProps = Pick<
+    UseEmojiPickerType,
+    | 'i18n'
+    | 'onMouseOver'
+    | 'onSelectEmoji'
+    | 'emojiLibrary'
+    | 'isSearching'
+    | 'searchResult'
+    | 'visibleCategories'
+    | 'refs'
+    | 'settings'
+>;
+
+export type EmojiButtonProps = {
+    index: number;
+    emoji: Emoji;
+    onSelect: (emoji: Emoji) => void;
+    onMouseOver: (emoji?: Emoji) => void;
+};
+
+export type RowOfButtonsProps = Pick<UseEmojiPickerType, 'onMouseOver' | 'onSelectEmoji' | 'emojiLibrary'> & {
+    row: GridRow;
+};
+
+const Button = memo(({ index, emoji, onSelect, onMouseOver }: EmojiButtonProps) => {
+    return (
+        <button
+            type="button"
+            aria-label={emoji?.skins[0]?.native}
+            tabIndex={-1}
+            data-index={index}
+            onClick={() => onSelect(emoji)}
+            onMouseEnter={() => onMouseOver(emoji)}
+            onMouseLeave={() => onMouseOver()}
+            className="tw-group/emojibutton tw-relative tw-flex tw-h-9 tw-w-9 tw-cursor-pointer tw-items-center tw-justify-center tw-border-none tw-bg-transparent tw-text-2xl tw-leading-none"
+        >
+            <div
+                aria-hidden="true"
+                className="tw-absolute tw-inset-0 tw-rounded-full tw-bg-[rgba(0,0,0,0.05)] tw-opacity-0 group-hover/emojibutton:tw-opacity-100"
+            />
+            <span data-emoji-set="native" className="tw-relative">
+                {emoji?.skins[0]?.native}
+            </span>
+        </button>
+    );
+});
+Button.displayName = 'Button';
+
+const RowOfButtons = memo(({ row, emojiLibrary, onSelectEmoji, onMouseOver }: RowOfButtonsProps) => (
```

Comment:
Is there much point in memoizing this? `onSelectEmoji` and `onMouseOver` trigger rerenders every time